authors/apps/articles/views.py

```diff
@@ -0,0 +1,124 @@
+import re
+import time
+
+import jwt
+from authors.apps.authentication.models import User
+from django.conf import settings
+from django.shortcuts import render
+from rest_framework import generics, permissions, status
+from rest_framework.exceptions import APIException
+from rest_framework.response import Response
+
+from .models import Article
+from .renderers import ArticlesRenderer
+from .serializers import ArticleSerializer
+
+
+class ArticlesListCreateAPIView(generics.ListCreateAPIView):
+    queryset = Article.objects.all()
+    serializer_class = ArticleSerializer
+    renderer_class = ArticlesRenderer
+    permission_class = permissions.IsAuthenticatedOrReadOnly
+
+    def create(self, request, *args, **kwargs):
+
+        article_ = request.data.get('article')
```

Comment:
Why does the article variable name have an underscore at the end ..?

Reply:
I was using a similar variable elsewhere, didnt want to confuse them

Reply:
Just rename it to something else.

Reply:
it actually didn't matter, it can have the same name "article".